org.spoofax.interpreter.core/src/main/java/org/spoofax/interpreter/library/ssl/SSL_immutable_set_strict_subset.java

```diff
@@ -0,0 +1,16 @@
+package org.spoofax.interpreter.library.ssl;
+
+import org.spoofax.interpreter.core.IContext;
+import org.spoofax.interpreter.stratego.Strategy;
+import org.spoofax.interpreter.terms.IStrategoTerm;
+
+public class SSL_immutable_set_strict_subset extends SSL_immutable_set_strict_subset_eq {
```

Comment:
Is there a reason that this is a separate primitive instead of an alias `set-strict-subset = set-strict-subset(id)` in Stratego? If for performance only, this is maybe premature optimization.

Reply:
I don't remember exactly why I started doing this, but probably premature optimisation. It's a pattern with other primitives as well, anything that ends with `_eq` (about 12 different primitives) have a version without `_eq` that passes `Object::equals`.

Reply:
Maybe @Gohla knows what the reason was for other primitives to have a specialized `equals` version?

Reply:
No, I wrote those other ones too, those are for the immutable set/map ^^